docs: add 3d-view endpoint doc

lniehaus opened the pull request.

jarkenau commented:
@Anafabula unified the api docs, in the last days, your docs don't follow the new structure. Every page now hast an overview section at the beginning, https://naturerobots.github.io/HSOS-SEP-PlantMap-2022/reference/bed-plant-endpoints/#overview. And the headers after that use a real name instead of the URL. Could you change that?

lniehaus commented:
> @Anafabula unified the api docs, in the last days, your docs don't follow the new structure. Every page now hast an overview section at the beginning, https://naturerobots.github.io/HSOS-SEP-PlantMap-2022/reference/bed-plant-endpoints/#overview. And the headers after that use a real name instead of the URL. Could you change that?

I reworked the 3d-view to contain an overview like described and added a storage system section

jarkenau commented on docs/reference/3d-view.md:
```diff
@@ -0,0 +1,72 @@
+Send a GET request to this endpoint to get the Plant information.
+The response contains two elements.
+Firstly an array called plants, which contains informations about every plant in the bed.
+Secondly an global elements which describes center position of the bed.
```
```suggestion
Secondly an global element which describes center position of the bed.
```

jarkenau commented on docs/reference/storage-system.md:
```diff
@@ -0,0 +1,35 @@
+## Basics
+
+To store files like the garden images, logs or plant pointclouds we use the storage folder.
+images is a common folder, which allows to store files that are not accessible through an url.
```
```suggestion
Images is a common folder, which allows to store files that are not accessible through an url.
```

jarkenau reviewed (commented):
typos
